plugins/hypervisors/vmware/src/main/java/com/cloud/hypervisor/vmware/manager/VmwareStorageManagerImpl.java

```diff
@@ -1113,6 +1113,9 @@ private long getVMSnapshotChainSize(VmwareContext context, VmwareHypervisorHost
         DatastoreMO dsMo = new DatastoreMO(context, morDs);
         HostDatastoreBrowserMO browserMo = dsMo.getHostDatastoreBrowserMO();
         String datastorePath = (new DatastoreFile(dsMo.getName(), vmName)).getPath();
+        if (datastorePath.startsWith("[-iqn.")) { // ex. [-iqn.2010-01.com.solidfire:3p53.data-9999.97-0] i-2-9999-VM
```

Comment:
Note: similar code exists in the caller that checks for the specific datastore string

Reply:
I'll attempt refactoring next week